examples/secrets/Earthfile

```diff
@@ -1,3 +1,4 @@
+VERSION 0.6
```

Comment:
Just double-checking that this was intentional.

A comment explaining why this example uses `0.6` would be useful here. I think if I were trying to use earthly and was browsing through the examples, I would be surprised to see old versions pinned with no explanation.